### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Sara Camassa, David Berger

### DISCUSSION THREAD

**Discussion by Davtek11** [heig-vd-dai-course/heig-vd-dai-course#460] [Show and tell]
Notre projet consiste en l'implémentation d'un démineur. Le serveur a connaissance de l'état actuel du jeu (emplacement des bombes, des drapeaux, etc), et le client peut interagir avec le jeu à travers différentes commandes:
- FLAG <case> : mettre (ou enlever) un drapeau sur une case donnée
- CHECK <case> : vérifier une case donnée
- QUIT : arrêter la connexion
Le serveur indique au client s'il a CHECK une case contenant une bombe (perdu), ou s'il a FLAG toutes les bombes (gagné).

Chaque connexion avec un client est indépendante, ce qui fait que chaque utilisateur joue à une partie séparée et n'interagit pas avec les autres utilisateurs.

Link to the repository : https://github.com/scamassa/DAI_lab02
Final Commit hash : e03dae00485b4c6d6d7754f852822809625b569c

**Comment by GeraudSilvestri** [comment]
The concept is good for me

Remember to start simple and add more complex features once the base is working as intended.
